test/NNODE2_tests.jl

```diff
@@ -0,0 +1,53 @@
+using Test, Flux
+using Plots
+using DiffEqDevTools
+using DiffEqBase, NeuralNetDiffEq
+
+# Homogeneous
+f(du, u, p, t) = -du+6*u
+tspan = (0.0f0, 2.0f0)
+u0 = [1.0f0]
+du0 = [0.0f0]
+dt = 1/20f0
+function an_sol(du0, u0, p, t)
+  [1/5 * exp(-3*t) *(3*exp(5*t)+2.0)]
+end
+prob = SecondOrderODEProblem(ODEFunction{false}(f, analytic=an_sol), u0, du0, tspan)
+chain = Chain(Dense(1,5,σ),Dense(5,1))
+opt = ADAM(1e-04, (0.9, 0.95))
+sol = solve(prob, NeuralNetDiffEq.NNODE2(chain,opt), dt=dt, verbose = true, abstol=1e-10, maxiters = 200)
+println(sol)
```

Comment:
should add a test here.

Reply:
for reference, see the NNODE tests. It checks a solution value against an analytical solution and tests that it's close enough.